src/lib/components/top-nav.svelte

```diff
@@ -69,6 +70,7 @@
         optionValueKey="namespace"
         on:change={handleNamespaceSelect}
         minSize={32}
+        href={routeForNamespace({ namespace })}
```

Comment:
Do we want to disable this if the Namespace doesn't exist?

| | |
|-|-|
|<img width="1115" alt="Screenshot 2024-05-21 at 2 28 20 PM" src="https://github.com/temporalio/ui/assets/15069288/c3819eca-2cac-46fe-957d-096ecf7a03da">|<img width="1127" alt="Screenshot 2024-05-21 at 2 27 34 PM" src="https://github.com/temporalio/ui/assets/15069288/5a9b63fd-54d9-4656-a4c6-c22284fa04c4">|

Reply:
🤌